src/platform/AT32/bus_spi_at32bsp.c

```diff
@@ -134,6 +134,8 @@ void spiInternalResetStream(dmaChannelDescriptor_t *descriptor)
     DMA_CLEAR_FLAG(descriptor, DMA_IT_HTIF | DMA_IT_TEIF | DMA_IT_TCIF);
 }
 
+#pragma GCC optimize "O0"
```

Comment:
Comment should explain this hack

And using `__attribute__ ((optimize ...` may be slightly better

Reply:
Resolved by https://github.com/betaflight/betaflight/pull/14070/commits/57760c38dcd6bd3c42a8e57184c3f8e011aa6695 This was an oversight.